lib/build-tokens.js

```diff
@@ -104,17 +123,25 @@ async function buildTokensCommand(commandArgs) {
           {
             format: 'css/custom-variables',
             destination: `themes/${themeVariant}/variables.css`,
-            filter: hasSourceTokensOnly ? 'isSource' : undefined,
+            filter: hasSourceTokensOnly
+              ? `isThemeVariant${themeVariant.charAt(0).toUpperCase()}${themeVariant.slice(1)}SourceOnly`
+              : `isThemeVariant${themeVariant.charAt(0).toUpperCase()}${themeVariant.slice(1)}`,
```

Comment:
[inform] Currently, the registered format `css/custom-variables` performs filtering within its own logic (i.e., filtering on `dictionary.allTokens`), rather than treating the filter as a legit style-dictionary filter here. New filters are now registered to filter tokens by the appropriate `themeVariant` file path within the `filter` property here versus within the definition of  `css/custom-variables`.